gh-1365 ignored luacheck warnings (#1382)
tarantool/doc: doc/dev_guide/lua_style_guide.rst

    end
    return nil, err

===========================================================
                    Luacheck
===========================================================

> To check the codestyle Tarantool uses ``luacheck``. It analyses different

Reviewer: ```suggestion
To check the codestyle, Tarantool uses ``luacheck``. It analyses different
```

> aspects of code, such as unused variables, and sometimes more then needed.

Reviewer: ```suggestion
aspects of code, such as unused variables, and sometimes it checks more aspects than needed.
```

> So there is an agreement to ignore some set of warnings generated by ``luacheck``:

Reviewer: ```suggestion
So there is an agreement to ignore some warnings generated by ``luacheck``:
```
